source/collections/Queue.ooc

```diff
@@ -12,17 +12,20 @@ Queue: abstract class <T> {
 	dequeue: abstract func -> (T, Bool)
 	peek: abstract func -> (T, Bool)
 	//TODO: This works much better and will do for now
-	dequeue: abstract func ~out (result: T*) -> Bool
+	dequeue: abstract func ~out (result: T*) -> Bool // Also crashing the compiler
```

Comment:
I suggest slowly obsoleting it over a cycle of pull requests.
Removing it right away might break something if someone can't come up with a good default value for a value type.